Add inconsistent-variable-name rule (#847)
MarketSquare/robotframework-robocop: docs/releasenotes/3.2.0.rst

            Log    ${varia ble}  # matches ${variable} but is not consistent
        END

New rule for overwriting reserved variables (#553)
--------------------------------------------------

Added new rule W0324 ``overwriting-reserved-variable`` which reports overwriting reserved Robot Framework variables.

> There are variables used by Robot Framework itself which are not safe to overwrite. See the list at

Reviewer: ```suggestion
There are variables used by Robot Framework itself which should not be overwritten. See the list at
```
